test/acceptance/test/add_tests.go

```diff
@@ -376,7 +376,8 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		workloadNamespace := "my-nginx-" + uniqueSuffix
 		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
 		wegoNamespace := "my-space"
-		addCommand := "app add . --branch=" + branchName + " --namespace=" + wegoNamespace
+		url := "ssh://git@github.com/" + os.Getenv("GITHUB_ORG") + "/" + appRepoName + ".git"
+		addCommand := "app add --url=" + url + " --branch=" + branchName + " --namespace=" + wegoNamespace
```

Comment:
a good practice in these cases where we need to create strings is to use `fmt.Sprint('app add --url=%s --branch=%s --namespace=%s,url,branchName,wegoNamespace)`

Reply:
Yes, we follow this practice in the majority of the functions. But for `add` functions, I feel like it provides better visibility as to which combination of `wego app add` flags we're using.